sos/plugins/block.py

```diff
@@ -69,10 +53,12 @@ def setup(self):
                     "fdisk -l %s" % disk_path
                 ])
 
-        lsblk_file = self.get_cmd_output_now("lsblk -f -a -l")
+        lsblk = self.get_cmd_output_now("lsblk -f -a -l")
         # for LUKS devices, collect cryptsetup luksDump
-        if lsblk_file:
-            for dev in self.get_luks_devices(lsblk_file):
-                self.add_cmd_output('cryptsetup luksDump /dev/%s' % dev)
+        if lsblk['status'] == 0:
```

Comment:
See my comment in `abrt` - ideally, we shall be coherent here (no strong prferences, either testing for zero status or skipping such a test is fine to me).

Reply:
I'm definitely in the "always check status == 0" camp.

We have no other way to gate against the iterations that follow from executing against error output when the command fails for one reason or another.